discover/portal/articles/210-setting-up/01-system-settings/01-system-settings.markdown

```diff
@@ -0,0 +1,121 @@
+
+![Figure 2: System Settings are organized by component.](../../../images/system-settings-nav-search.png)
+
+Changing an application's default configuration options in System Settings isn't
+difficult. Once you find what you're looking for, simply click the name of the
+component you want to configure, or click the actions button
+(![Actions](../../../images/icon-actions.png)) then click *Edit*. Make any
+changes you'd like, then click *Save*. Your configuration changes are saved and
+applied throughout the system. 
+
+![Figure 3: After saving changes to a configuration, the actions *Reset Default Values* and *Export* are available for it.](../../../images/system-settings-actions.png)
+
+Keep in mind that some configurations are available at more than one scope. Once a
+configuration change is made at a more granular scope, making a change at the
+system level won't do anything. 
+
+For example, whether to allow comments on Blogs Entries is configurable. Set the default behavior is set at Control Panel &rarr; Configuration &rarr; System Settings &rarr; Blogs. In the Blogs entry under Widget Scope, disable the *Enable Comments* checkbox. 
+
+Now add a Blog Entry to a site's Content &rarr; Blogs application. Then go to a
+public page and add the Blogs Widget to the page. Click the Actions button
+(![Actions](../../../images/icon-actions.png)) for the Widget and select
+*Configuration*. You'll see the same Enable Comments checkbox, and its default
+is now false (unchecked). Checking the box in the Widget Configuration screen
+breaks its link with the System Settings entry. Changing the System
+Settings configuration does nothing to this Widget Instance anymore.
+
+If you make some regrettable configuration decisions and can't recall exactly
+what you did, start over by clicking the actions button
+(![Actions](../../../images/icon-actions.png)), then clicking *Reset Default
+Values*. 
+
+## Configuration Scope [](id=configuration-scope)
+
+While browsing the categories of System Settings, you'll notice that clicking
+into a topic (for example, Blogs) reveals entries at different scopes.each entry
+has a *Name* and a *Scope*. This scope is different than the overall
+application configuration scope discussed in the introduction to this section.
+This is the System Settings application after all, so all of the configuration
+done here affects the System scope. The Scope field here refers to the scope at 
+which this default, system-wide configuration is overridden. There are four 
+values that you'll see under Scope: 
+
+- *System:* Any configuration made at the system scope here becomes the final
+  value for the application in a system-wide fashion. It affects the whole
+  system and isn't overridden anywhere else. 
+
+    ![Figure 4: Some System Settings entries have a System scope.](../../../images/system-settings-system-scope.png)
+
+- *Virtual Instance Scope:* Making configuration changes at this level affects
+    the entire Virtual Instance. 
```

Comment:
Hmm, what is configured within System Settings will be the default for all Virtual Instances.

Reply:
Good catch, that's definitely wrong. I want to say it's overridden in Instance Settings, but I'm not convinced that this is consistent. But I cannot find many Virtual Instance Scope settings to test this out with. So far I have only seen one for Web Form, and I'm sure that's not the example I should look at.